lib/widgets/layouts/adaptive_layout/adaptive_scaffold_route.dart

```diff
@@ -19,13 +20,14 @@ class AdaptiveScaffoldRoute extends StatelessWidget {
     Key? key,
     required this.body,
     this.secondaryBody,
+    this.displayAppBar = true,
   }) : super(key: key ?? scaffoldWithNestedNavigationKey);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       backgroundColor: Theme.of(context).colorScheme.surface,
-      appBar: const AdaptiveScaffoldAppBar(),
+      appBar: displayAppBar == true ? const AdaptiveScaffoldAppBar() : null,
       body: AdaptiveLayout(
         internalAnimations: false,
         body: SlotLayout(
```

Comment:
`displayAppBar ? const AdaptiveScaffoldAppBar() : null`

Reply:
```
 final bool? displayAppBar;
```

Try checking that the value isn't 'null' before using it as a condition.